src/aws-util/src/s3.rs

```diff
@@ -8,9 +8,11 @@
 // by the Apache License, Version 2.0.
 
 use aws_sdk_s3::config::Builder;
-use aws_sdk_s3::Client;
 use aws_types::sdk_config::SdkConfig;
 
+pub use aws_sdk_s3::Client;
+use bytes::Bytes;
```

Comment:
I think this is meant to go in the other import block?

Reply:
ahh yeah I should move this up, thanks!